docs/athenz_templates.md

```diff
@@ -0,0 +1,235 @@
+# Introducing Athenz Templates
+Templates are a collection of predefined roles, 
+policies and services that can be applied on a domain. 
+Imagine in a real world scenario if you are creating a solution
+for any organization(Openstack, Kubernetes) 
+and subsequently want to set up provisioning rules(a static set of 
+roles and policies) for the solution, rather than requiring each domain 
+admin to create each of these roles and policies manually, 
+you can pre-bundle the roles and policies as a template and Athenz 
+will automatically apply them for you while creating or updating a domain.
+    
+## Templates
+
+### Creating and updating templates
+Templates today exists in json format in git repo under a file called 
```

Comment:
Remove the reference to git repo. We should say that the system admin will manage all the templates in a file their data repository.